lib/console.js

```diff
@@ -139,12 +139,14 @@ Console.prototype.dir = function dir(object, options) {
 };
 
 
-Console.prototype.time = function time(label) {
+Console.prototype.time = function time(label = 'default') {
+  label = `${label}`;
```

Comment:
Yep, IDL rules are weird. Before ES6 you could also use `label + ''` but not anymore due to some `@@toPrimitive` semantics that wouldn't work with Dates.